protobuf/kubearmor.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.30.0
-// 	protoc        v4.23.4
+// 	protoc-gen-go v1.26.0
+// 	protoc        v3.19.4
```

Comment:
Can you use latest protoc? that will avoid these changes latest protoc seems to be v4.23.4